rfcs/20210403-DMF-v1.0.0.md

```diff
@@ -0,0 +1,122 @@
+
+# nanoMFG Software Planning Document
+<!-- Replace text below with long title of project:short-name -->
+## nanoMFG Data Management Framework
+### Target Release: #.#.# : Month Day, 20##
+
+## Project Team
+<!-- Complete table for all team members 
+ roles: lead, developer, reviewer, contributer
+ status: active, inactive
+-->
+Name | Role | github user | nanohub user | email | status
+---|---|---|---|---|---
+Joe Shmo | lead | jshmo | jshmo | jshmo@nanocool.co | active
+Alice Land | developer | alice1 | alice2 | alice@wonderland | inactive
+
+**nanoMFG Github Team(s):** @coolteam
+**nanoHUB Group(s):**
+
+## 1. Introduction
+
+
+### 1.1 Purpose and Vision Statement
+<!-- Why are we building this tool?
+What is the key benefit
+How does it relate to existing tools and existing software?
+How does it fit into the overall objectives for the nano **manufacturing** node?
+Who will use this software?
+-->
+The first version of the DMf is intended to provide experimental labs working with 2D materials with framework to describe, catalog, analyse and publish data.
+
+##### Relationship to Existing Software
+Seems there are a number of commertional options for scientific data management software (SDMS)
+
+##### Our Approach
+
+### 1.2 References
+<!--List any documents or background material that are relevant.  Links are useful. For instance, a link to a wiki or readme page in the project repository, or link to a uploaded file (doc, pdf, ppt, etc.).-->
```

Comment:
is a section needed for this, or would these references be in-line in previous subsections?